assets/dataloggers.csv

```diff
@@ -374,6 +374,7 @@ Kinemetrics,Q330HR/6,4004,13957,
 Kinemetrics,Q330HR/6,4008,13958,
 Kinemetrics,Q330HR/6,4009,13959,
 Kinemetrics,Q330HR/6,4010,13960,
+Kinemetrics,Q330HR/6,4226,,
```

Comment:
We shall clearly indiacte that this is not a GeoNet asset 
like
  "IU: Global Seismograph Network (GSN - IRIS/USGS) and https://doi.org/10.7914/SN/IU"